portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,28 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+// Show current date and time on every page
+function showDate() {
+    const date = new Date();
+    const dateContainer = document.getElementById("date");
+    dateContainer.style.color = "white";
+    dateContainer.innerHTML = date;
+} 
+
+function displayExperience() {
+    document.getElementById("TravelExp").classList.toggle("show");
+}
+
+window.onclick = function(event) {
+  if (!event.target.matches('.dropbutton')) {
```

Comment:
This is a good place to add a clarifying comment. When I first looked it over, I was asking "What's the goal of this code? Why are we not matching .dropbutton?"

Something simple like "Remove all dropdowns when user clicks outside the map image" would be perfect.